dev/com.ibm.ws.artifact.zip/src/com/ibm/ws/artifact/zip/internal/ZipFileContainer.java

```diff
@@ -1167,13 +1167,36 @@ public ZipFileEntry getEntry(String entryPath, boolean normalized) {
         }
     }
 
+    @Trivial 
+    URL createSharedClassCacheURL(String r_entryPath) {
+        File useArchiveFile = getArchiveFile();
```

Comment:
`useArchiveFile` sounds more like a boolean (i.e. used to determine if we are using an archive file or not).  Would `archiveFile` be a better variable name?

Reply:
'archiveFile' is already used as an instance variable name.  'use' is added to prevent a compiler warning.
I've been putting 'use' as a prefix, but other values, say, 'my' or 'a' would work just as well.

Reply:
This code is now gone so it is a moot point.